lib/push.js

```diff
@@ -177,6 +181,39 @@ module.exports = function (log, db) {
     },
 
     /**
+     * Notifies every device in the account except the current one that the
+     * password was changed
```

Comment:
I could check session/status browser side, I'll come back with corrections to this patch